admin/views/pupil-pin/generate-pins-overview.ejs

```diff
@@ -13,18 +13,18 @@
             </div>
         <% } %>
             <h1 class="govuk-heading-xl"><%= pinEnv === 'live' ?
-                        'Generate school passwords and PINs for the official check' : 'Generate passwords and PINs for the try it out check'
+                        'Generate school password and PINs for the official check' : 'Generate school password and PINs for the try it out check'
                 %>
             </h1>
             <%- partial('partials/_gds_session_expiry') %>
             <p class="govuk-body">
-                Pupils will need a PIN and school password to <%= pinEnv === 'live' ? 'start the MTC' : 'sign into the try it out area' %>.
-                The link for pupils to <%= pinEnv === 'live' ? 'start the check' : 'try it out' %> is:
-                <a class="govuk-link" target="_blank" rel="noopener noreferrer" href="<%= pupilAppURL %>"><%= pupilAppURL %></a>
-                <div class="govuk-body">These expire at 4pm daily.</div>
+                Pupils will need a school password and individual PIN to sign into the [TODO:CHECK_TYPE] check.' %>.
```

Comment:
Flagging TODO item, for a later review, when this PR is finalised.

Reply:
thanks, i've now removed this! :)